### PR TITLE
Main

### PR DESCRIPTION
**Pull request by delvor**
<!--  Code Review Checklist

  - [ ] Markdown code is correct well-formatted
  - [ ] Spelling and grammar are ok

### CODE REVIEW

**Comment by tvsirius** [comment]
def do_some(number:int, word:str, variable:MyClass):
      # do something
      
I suggest we also use type notaion for result
         
          def do_some(number:int, word:str, variable:MyClass)->[str]:
                  # do something
      
      
Agree?

File: group_study/constraints.md [hunk 1]
```diff
@@ -2,7 +2,43 @@
...
+
+    # func do_some has a parameter of integer and string, and MyClass
+    def do_some(number, word, variable):
+      # do something
+    
+    If we will leave it as is someone could pass to number say float, and program will throw an error, and it will require some debugging, etc.
+
+    This is why declaring func parameters we should notate types of parameters like that
+    
+    def do_some(number:int, word:str, variable:MyClass):
```
Comment (tvsirius) [inline group_study/constraints.md]: I suggest we also add a type notaion for result
         def do_some(number:int, word:str, variable:MyClass)->[str]:

**Review by delvor — COMMENTED** [review]
>     def do_some(number:int, word:str, variable:MyClass):
 >     # do something
      
> I suggest we also use type notaion for result
>         
>          def do_some(number:int, word:str, variable:MyClass)->[str]:
 >                 # do something
      
      
> Agree?     
 

This even better. 
Made changes

**Review by tvsirius — APPROVED** [review]
Good!

**Pull request merged**